packages/terra-form-select/src/_Menu.jsx

```diff
@@ -18,10 +20,18 @@ const propTypes = {
    * The content of the menu.
    */
   children: PropTypes.node,
+  /**
```

Comment:
We'll want to add `@private` to the description for this prop, it doesn't need to be publically documented and this string will prevent it from being rendered on the docsite.

```
/**
 * @private Translation object to use for interpreting strings
 */
```

Reply:
Done. https://github.com/cerner/terra-core/pull/2163/commits/207f487d49ce1516167e38a806510a7584848084